### DISCUSSION TITLE
It would be good were Conky to read the CPU temperature of (a greater number of?) AMD chips

### DISCUSSION THREAD

**Discussion by LinuxOnTheDesktop** [brndnmtthws/conky#2050] [Ideas]
[This thread elsewhere](https://forums.debian.net/viewtopic.php?t=74938) reveals that Conky's `acpitemp` cannot get the temperature of a Phenom2 quad-core chip. I find that similarly Conky cannot get the temperature on my new Framework 13-AMD, with a Ryzen 6 764OU [EDIT: no, not Ryzen 6; rather, Ryzen _5_] processor. (See perhaps also [this further report](https://bbs.archlinux.org/viewtopic.php?id=32400).)

One can write an external program to allow Conky to display the values on such AMD chips, but native Conky support would be nice!

**Comment by Caellian** [comment]
Moved to #2051
